tar/test/test_list_item.c

```diff
@@ -81,8 +86,7 @@ run_tvf(void)
 	    systemf("%s tvf test_list_item.tar >tvf.out 2>tvf.err", testprog));
 
 	/* Restore the original date formatting. */
-	if (orig_lc_time != NULL)
-		set_lc_time(orig_lc_time);
+	set_lc_time(orig_lc_time);
```

Comment:
At the moment, this commit is doing 2 things:
- fix `make test` on some environments by changing `en_US` to `C`.  Great change!
- make `set_lc_time` unset `LC_TIME` if it wasn't originally set.  As far as I know, `setenv()` isn't supposed to change anything other than the current process, so this wouldn't do anything for `make test`.

I personally would put the `set_lc_time()` changes in a different commit; call that other commit something like "test_list_item.c: unset LC_TIME if it wasn't originally set".

Reply:
I'm fine with splitting these into two commits. Done. But since both are required to fix the issue reported, they have both the "Fixes" line in their commit messages.

Reply:
> The same process is responsible for the other tests as well.

Ah, I didn't realize that.  (Or maybe that's why I attempted to reset `LC_TIME` in the first place, but I unfortunately didn't include that info in the comment?)